account_financial_report_qweb/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'QWeb Financial Reports',
-    'version': '9.0.1.0.2',
+    'version': '9.0.1.0.3',
```

Comment:
now is 9.0.1.0.3 if you rebase. So it should be 9.0.1.0.4 or 9.0.1.1.0

Reply:
I updated the version number in the Github UI